HomeWork_18/HW18.py

```diff
@@ -0,0 +1,35 @@
+class Bot:
+    def __init__(self, name, message):
+        self.name = name
+        self.message = message
+    def send_message(self):
+        print(f"{self.name} say {self.message}")
+
+    def say_name(self):
+        print(f"My name is {self.name}")
+
+n = Bot("Aleksandr", "I'm study")
+m = Bot("Hello", "How are you?")
+n.say_name()
+m.send_message()
+
+
+class TelegramBot(Bot):
+    def __init__(self,url,chat_id):
+        self.url = url
+        self.chat_id = chat_id
+    def set_url(self):
+        print(f"my url : {self.url}")
+    def set_chat_id(self):
+        print(f"My ID is: {self.chat_id}")
+
+    def send_message(self):
+        super.__init__(self.message,self.name)
```

Comment:
It is wrong to call a constructor of parent outside a constructor of the current class